spec.bs

```diff
@@ -3610,6 +3618,13 @@ A <dfn>reporting context</dfn> is a [=struct=] with the following [=struct/items
     contribution entries=].
   : <dfn>private aggregation allowed</dfn>
   :: A [=boolean=], initially false.
+  : <dfn>local leader info</dfn>
+  :: A [=leading bid info=], describing information on what, if anything, won this particular
+    subauction.

```

Comment:
Well, it can be either component auction or top-level auction. 
...This also gets into the whole "Process the Private Aggregation contributions for an auction tree" thing -- maybe I could rename the method that calls to talk about subauctions, and the main thing can just be "Process the Private Aggregation contributions"?

Reply:
Yeah that would be better.
And "subauction" is confusing, since I would think it's component auction. You meant "unnested" auction (component, or single seller auction)?

Reply:
No. It can be component, it can be single-seller, it can be top-level.

Reply:
oh then "sub" confused me more.

Reply:
OK, dropped the sub, and renamed the other methods.